recipes/sdl_image/all/conanfile.py

```diff
@@ -119,11 +119,6 @@ def generate(self):
         tc.variables["XCF"] = self.options.xcf
         tc.variables["XPM"] = self.options.xpm
         tc.variables["XV"] = self.options.xv
-        # TODO: https://github.com/bincrafters/community/pull/1317#pullrequestreview-584847138
```

Comment:
+1 it should be removed given the converstation

Reply:
so there should be more options, like you can link statically, link dynamically or use `dlopen`/`LoadLibrary` to locate a specific image plug-in.

Reply:
there was https://github.com/conan-io/conan-center-index/pull/14493